src/modules/AutoinstStorage.rb

```diff
@@ -165,6 +165,7 @@ def set_devices(storage_config)
 
           next { dev => drive }
         end
+        {}
```

Comment:
Why returning a hash? According to the method's description, the return value is not important.

Reply:
The target map will be returned. The docu is wrong. I have fixed it.